MaterialLibrary/src/main/java/io/doist/material/drawable/TintDrawable.java

```diff
@@ -83,6 +83,11 @@ public boolean isStateful() {
     }
 
     @Override
+    public boolean setState(int[] stateSet) {
+        return super.setState(stateSet) | super.superSetState(getState());
```

Comment:
Pinging @jpmcosta. I honestly don't remember this change, but I recall getting help from João because of the `WrapperDrawable` update. There was indeed a problem that we worked around by doing this.

Will remove the unnecessary semicolon :+1: